workflow-runtime/src/main/java/com/squareup/workflow/internal/SubtreeManager.kt

```diff
@@ -95,10 +95,10 @@ import kotlin.coroutines.EmptyCoroutineContext
  * @param snapshotCache
  */
 @OptIn(ExperimentalWorkflowApi::class)
-internal class SubtreeManager<StateT, OutputT : Any>(
+internal class SubtreeManager<StateT, OutputT>(
   snapshotCache: Map<WorkflowNodeId, TreeSnapshot>,
   private val contextForChildren: CoroutineContext,
-  private val emitActionToParent: (WorkflowAction<StateT, OutputT>) -> Any?,
+  private val emitActionToParent: (WorkflowAction<StateT, OutputT>) -> MaybeOutput<Any?>,
```

Comment:
Do we really need both `MaybeOutput` and `OutputHolder`? No strong feelings about it, just surprised.

Reply:
They're both internal to their modules. To share them, we'd have to make one of them public, and I don't want to do that.